src/app/core/services/search-registration/offline-capable-search-service.ts

```diff
@@ -100,7 +100,12 @@ export class OfflineCapableSearchService extends SearchService {
 
     // Update the last fetched time shown in filter menu whenever last fetched in offline db changes
     this.lastSyncTime$
-      .pipe(map((lastFetchedMs) => new Date(lastFetchedMs)))
+      .pipe(
+        map((lastFetchedMs) => {
+          if (lastFetchedMs == 0) return '-';
```

Comment:
Jeg mener denne bør returnere null eller undefined i stedet for '-'. Er det ikke det som er vanlig for en variabel som ikke er satt?